homeassistant/components/tado/climate.py

```diff
@@ -399,7 +399,7 @@ def set_temp_offset(self, offset):
         """Set offset on the entity."""
 
         _LOGGER.debug(
-            "Setting temperature offset for device %s setting to (%d)",
+            "Setting temperature offset for device %s setting to (%.2f)",
```

Comment:
```suggestion
            "Setting temperature offset for device %s setting to (%.1f)",
```

At least in the Tado app, offsets seem to support only 1 digit after the comma.